scripts/schemaGenerator.py

```
@@ -55,6 +58,37 @@ def get_options_list_for_enum(field):
+    if "configKey" in field:
+        configKey = field["configKey"]
+    isDynamicConfigSupportedField = configKey in dynamicConfigSupported
+    return isDynamicConfigSupportedField
+
+
+def generate_uiconfig_pattern(field, dbConfig=None) -> str:
+    field_supports_dynamic_config = is_dynamic_config_supported_field(field, dbConfig)
+    destName = ""
+    if "name" in dbConfig:
+        destName = dbConfig["name"].lower()
+    configKey = field["configKey"]
```

Ruff (0.8.2)

Local variable `configKey` is assigned to but never used

Remove assignment to unused variable `configKey`

(F841)